code/svb-docker/python/svb_web/core/views/internal/populate_db.py

```diff
@@ -0,0 +1,52 @@
+'''
+WARNING: hacky testing purposes only
+please remove this as soon as possible.
+'''
+from django.http import HttpResponse
+
+def populate_db(request):
+    #############################
+    from core.models import Customer
+    customer_one = Customer(
+        first_name="first name 1",
+        costume="costume 1",
+        referrer=None,
+        security_candy="security candy 1",
+    )
+    customer_one.save()
+    customer_two = Customer(
+        first_name="first name 2",
+        costume="costume 2",
+        referrer=customer_one,
+        security_candy="security candy 2",
+    )
+    customer_two.save()
```

Comment:
NP: Might be nice to make this parametric like;
```suggestion
    CUSTOMER_IDS = [1, 2]
    customer_models = []
    for customer_id in CUSTOMER_IDS:
        customer = Customer(
            first_name=f"first name {customer_id }",
            costume=f"costume {customer_id}",
            referrer=None,
            security_candy=f"security candy {customer_id}",
        )
        customer.save()
        customer_models.append(customer)
```